common/include/verilog_int.h

```diff
@@ -144,31 +144,68 @@ struct vint {
 	//////////////////////
 	// comparison
 	//////////////////////
-	// TODO > < >= <= !=
-	bool operator==(const vint& rhs) const {
-		return ::std::equal(::std::begin(v), ::std::end(v), ::std::begin(rhs.v));
+
+	// compare() will return:
+	// 1: larger
+	// 0: must check further
+	// -1: smaller
+	int compare(const vint& rhs) const {
+		if constexpr (num_word > 1) {
+			for (unsigned i = num_word; i > 0;) {
+				--i;
+				if (v[i] > rhs.v[i]) {
+					return -1;
```

Comment:
if v[i] > rhs.v[i], why it will return -1? I think -1 means smaller?

Reply:
That's why we need unittest